### PR TITLE
[#87583206] allow set different height/width for thumbnail generation

### PR DESCRIPTION
**Pull request by FangAtKabam**
relates to #83452334
add functionality that allows set different height/width for thumbnail generation on ngImgCrop

### CODE REVIEW

File: compile/unminified/ng-img-crop.js [hunk 1]
```diff
@@ -437,12 +447,12 @@ crop.factory('cropArea', ['cropCanvas', function(CropCanvas) {
   CropArea.prototype._dontDragOutside=function() {
     var h=this._ctx.canvas.height,
```
Comment (wilatkabam) [inline compile/unminified/ng-img-crop.js]: Maybe rename h and w to height and width?

**Comment by jaygeeseman** [comment]
Could we get a working demo?

**Comment by jaygeeseman** [comment]
Or maybe just see a working demo on your computer. I just want to make sure it works as expected. It's not obvious from the code changes and your gatsby PR isn't updated to use this fork yet

**Comment by FangAtKabam** [comment]
http://fiddle.jshell.net/oq9kj7qo/

**Comment by FangAtKabam** [comment]
I put it here http://jsfiddle.net/sfang10/m4aas6yv/

**Comment by FangAtKabam** [comment]
the bug should be fixed

**Comment by FangAtKabam** [comment]
http://jsfiddle.net/sfang10/m4aas6yv/8/

**Comment by jaygeeseman** [comment]
This works well until the image is taller than it is wide, then it has the same problem where the rectangle shape changes vertically if you try to make it bigger than the image.

**Pull request merged**